index.js

```diff
@@ -20,5 +20,16 @@ api.get('/', (req, res) => {
   })
 })
 
+class Car {
+  constructor(name, year) {
+    this.name = name;
+    this.year = year;
+  }
+}
+
+const myCar1 = new Car("Ford", 2014);
+const myCar2 = new Car("Audi", 2019);
+
 api.listen(PORT, HOST)
 console.log(`Running on http://${HOST}:${PORT}`)
+
```

Comment:
It seems like it can offer feedback on more robust contecepts like class instances. It also seems like it's able to retain memory across the different commits that attribute to the broader PR change set between the branch and base.